index.html

```diff
@@ -1,56 +1,307 @@
+                            <li class="skill-item">Codepen</li>
+                            <li class="skill-item">Gitlab</li>
+                            <li class="skill-item">Terminal</li>
+                        </ul>
+                 </li>
+          </ul>
+        </div>
+    </section><!-- #About me -->
+            <!-- ================
+              contact-me
+             ================ 
+    <section class="contact-me">
+        <div class="contact-info">
+            <h6 class="first-par">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h6>
+    
+          <form id="form" action="https://formspree.io/f/moqrjalz" method="post">
+                <input name="fullname" type="text" placeholder="Full name" minlength="5"
+                maxlength="50"
+                aria-label=" full name"
+                required>
+                <hr class="hr">
+        
+                <input
+                    name="email"
+                    id="email"
+                    class="stored"
+                    type="email"
+                    placeholder="Email address"
+                    aria-label=" Email Address"
+                    required>
+                    <hr class="hr">
+
+
+                    <textarea
+                    name="message"
+                    id="message"
+                    class="stored"
+                    minlength="5"
+                    maxlength="500"
+                    aria-label=" Comments"
+                    required
+                    placeholder="Enter text here"
+                  ></textarea>
+                  <button id="submit" type="submit">Get in touch</button>
+            </form>
+            </div>
+
+    </section> <!-- #contact-me -->
+    <!-- ================
+          Footer
+      ================ -->
+       <!-- <footer class="footer">
+           <div class="social-icon">
+            <ul class="social_media">
+                <li>
+                    <a href="https://github.com/ZuhRa-HashiMi" target="blank"><img src="assets/github.svg" alt="github"></a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/in/zuhra-hashimi-601966214/" target="blank"><img src="assets/Linkedin.svg" alt="linkedin"></a>
+                </li>
+             
+                <li>
+                    <a href="https://twitter.com/ZuhraHashimi4" target="blank"><img src="assets/twitter.svg" alt="twitter"></a>
+                </li>
+                <li>
+                    <a href="http://medium.com" target="blank"><img src="assets/medium.svg" alt="linkedin"></a>
+                </li>
+                <li>
+                    <a href="http://angellist.com" target="blank"><img src="assets/angellist.svg" alt="angel-list"></a>
+                </li>
+         
+            </ul>
+            <div class="shape"></div>
+            
+           </div> 
+       </footer>-->
+
```

Comment:
- [ ] Please kindly delete the comments lines of code to have a clean and professional code base. If you will need this code in the future you will just check this commit and you will find all the code 😉